lib/spire/error.rb

```diff
@@ -2,9 +2,10 @@ module Spire
   class Error < StandardError
     attr_reader :status_code
 
-    def initialize(message, status_code=nil)
+    def initialize(message, status_code = nil)
       @status_code = status_code
+      @message = message
```

Comment:
Now we can catch errors and call `exception.message` to get the error. Before we could only get the status code

Reply:
@caseyli Just tested this on develop with existing spire gem and you're right. Ill delete this PR